src/metabase/models/channel.clj

```diff
@@ -28,10 +28,18 @@
   {:type    mi/transform-keyword
    :details mi/transform-encrypted-json})
 
-(defn keywordize-type
-  "Ensure that `channel-type` is a keyword with the namespace `channel`."
-  [channel-type]
-  (if (and (keyword? channel-type)
-           (= "channel" (namespace channel-type)))
-    channel-type
-    (keyword "channel" (name channel-type))))
+(defn- assert-channel-type
+  [{channel-type :type}]
+  (when-not (= "channel" (-> channel-type keyword namespace))
+    (throw (ex-info "Channel type must be a namespaced keyword like :channel/http" {:status-code  400
+                                                                                    :channel-type channel-type}))))
+
```

Comment:
Extra blank line here

Reply:
there is only one newline on 36

Reply:
Ditto not seeing the rest of the map on my phone 🤦